api/invite.go

```diff
@@ -180,6 +180,58 @@ func (a *Api) GetReceivedInvitations(res http.ResponseWriter, req *http.Request,
 	}
 }
 
+// @Summary Get patient pending invite for a given team
+// @Description Get patient pending invite for a given team.
+// @ID hydrophone-api-GetPatientTeamPendingInvite
+// @Accept  json
+// @Produce  json
+// @Param teamId path string true "team id"
+// @Param patientId path string true "patient id"
+// @Success 200 {array} models.Confirmation
+// @Failure 400 {object} status.Status "teamId or patientId was not provided"
+// @Failure 401 {object} status.Status "Authorization token is missing or does not provided sufficient privileges"
+// @Failure 403 {object} status.Status "Authorization token is invalid"
+// @Failure 404 {object} status.Status "no invitations found for this user"
+// @Failure 500 {object} status.Status "Error while extracting the data"
+// @Router /teams/{teamId}/patients/{patientId}/invite [get]
+// @security TidepoolAuth
+func (a *Api) GetPatientTeamPendingInvite(res http.ResponseWriter, req *http.Request, vars map[string]string) {
+	tokenValue := getSessionToken(req)
+	token := a.token(res, req)
+	if token == nil {
+		return
+	}
+
+	teamID := vars["teamId"]
+	patientID := vars["patientId"]
+
+	_, _, err := a.getTeamForUser(nil, tokenValue, teamID, token.UserId, res)
```

Comment:
We should use the context if available
```suggestion
	_, _, err := a.getTeamForUser(req.Context(), tokenValue, teamID, token.UserId, res)
```